cg/meta/workflow/analysis.py

```diff
@@ -82,6 +83,14 @@ def verify_case_config_file_exists(self, case_id: str, dry_run: bool = False) ->
         if not Path(self.get_case_config_path(case_id=case_id)).exists() and not dry_run:
             raise CgError(f"No config file found for case {case_id}")
 
+    def get_validated_case(self, case_id: str) -> Case:
+        case: Case = self.status_db.get_case_by_internal_id(internal_id=case_id)
+        if not case.links:
+            raise CgError(f"No samples linked to {case_id}")
+        if nlinks := len(case.links) > 1 and not self.is_multiple_samples_allowed:
```

Comment:
`is_multiple_samples_allowed` is not defined for Balsamic or MIP, we need to move it from the nextflow analysis API here